### DISCUSSION TITLE
Using a custom date picker component causes enable backdrop to fail

### DISCUSSION THREAD

**Discussion by NFurkanSahin** [hakimio/ngx-ui-tour#132] [Q&A] [answered]
<img width="1050" alt="image" src="https://github.com/hakimio/ngx-ui-tour/assets/42929533/19d87412-8799-406f-96b9-10b83f5ae61a">

Using a datepicker component causes the pattern above. 
Can it be solved ? 

I've tried turning of view encapsulation but it didnt work.

**Comment by hakimio** [comment, marked as answer]
You can try to use `backdropConfig.zIndex` to solve this:
```js
{
    backdropConfig: {
        zIndex: '9999'
    }
}
```

**Reply by NFurkanSahin** [reply]
Thanks for the fast reply :D It worked quite well.
